pkg/telephone/telephone/telephone.go

```diff
@@ -0,0 +1,62 @@
+package telephone
```

Comment:
So close.  You have one too many telephones here.  The file should be at `pkg/telephone/telephone.go`.